tests/test_runner/tests.py

```diff
@@ -359,3 +359,22 @@ def test_empty_default_database(self):
         connection = testcases.connections[db.utils.DEFAULT_DB_ALIAS]
         self.assertEqual(connection.settings_dict['ENGINE'], 'django.db.backends.dummy')
         connections_support_transactions()
+
+
+class TestDatabaseMirror(unittest.TestCase):
```

Comment:
I think this could go in `test_utils` -- you can call the `setup_databases()` function rather than instantiating `DiscoverRunner`.